src/server/models/match.model.js

```diff
@@ -2,22 +2,24 @@ import mongoose from 'mongoose';
 const Schema = mongoose.Schema;
 
 let matchSchema = new Schema({
-  mentorId: Schema.Types.ObjectId,
-  menteeId: Schema.Types.ObjectId,
-  requestDate: {
+  mentor_id: {
+    type: Schema.Types.ObjectId,
+    required: true,
+  },
+  mentee_id: {
+    type: Schema.Types.ObjectId,
+    required: true,
+  },
+  request_date: {
     type: Date,
     default: Date.now(),
   },
-  isAccepted: {
-    type: Boolean,
-    default: false,
-  },
-  acceptDate: Date,
-  isCancelled: {
-    type: Boolean,
-    default: false,
+  //status of match {pending:0, accepted:1, rejected:2}
+  status: {
+    type: Number,
+    default: 2,
   },
-  cancelDate: Date,
+  response_date: Date,
```

Comment:
time of mentee's request -> request_date,
time of mentor's response -> response_date,